### ISSUE TITLE
🚀 Feature: more accurate email validation at sign in

### ISSUE THREAD

**Issue by donni106** [appwrite/appwrite#4963]
### 🔖 Feature description

The sign in form validates the email address:

<img width="526" alt="image" src="https://user-images.githubusercontent.com/1942953/211032318-77b7a7bc-76c5-4cb6-bef8-3ea492a02106.png">

When passing without the ending, i.e. ".com", the validation is missing and the form is sent to the server, which then validates:

<img width="662" alt="image" src="https://user-images.githubusercontent.com/1942953/211032443-eede5612-6ec9-4fb2-8ce1-c531982d7b18.png">

The email input could already validate against the full accurate email format.

### 🎤 Pitch

As the email format will be validates after sending the form anyways and there exists a form input validation in the frontend, it could be extended with the full email format validation including ".com" endings.

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment]
@TorstenDittmann, do we want to update the client-side validation for this? According to [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email), something@something is valid:

> A single properly-formed email address. This doesn't necessarily mean the email address exists, but it is at least formatted correctly. In simple terms, this means username@domain or username@domain.tld.

**Comment by shishir-prasad** [comment]
Hi,
I can work on this issue.

**Comment by AkashSharma001** [comment]
I would like to work on this.
@stnguyen90

**Comment by donni106** [comment]
Anyways I suggest having the same validations in the client and on the server?!

**Comment by Nandini99-git** [comment]
It looks like you are sign in for the first time. Follow this step
1. click to sign-in now
2. accept the certificate 
 and now try to sign up
I hope this will help.

**Comment by TorstenDittmann** [comment]
> @TorstenDittmann, do we want to update the client-side validation for this? According to [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email), something@something is valid:
> 
> > A single properly-formed email address. This doesn't necessarily mean the email address exists, but it is at least formatted correctly. In simple terms, this means username@domain or [username@domain.tld](mailto:username@domain.tld).

Yeah, let's allow these e-mails. Especially in Windows domain and co its quite common to have these type of e-mails.
